examples/tera_template/src/main.rs

```diff
@@ -0,0 +1,74 @@
+//! A Hello World example application for working with Gotham.
+
+extern crate gotham;
+extern crate hyper;
+extern crate mime;
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate tera;
+
+use hyper::{Response, StatusCode};
+
+use gotham::helpers::http::response::create_response;
+use gotham::state::State;
+use tera::{Context, Tera};
+
+///Assuming the Rust file is at the same level as the templates folder
+///, we can get a Tera instance that way:
+lazy_static! {
+    pub static ref TERA: Tera = {
+        compile_templates!(concat!(env!("CARGO_MANIFEST_DIR"), "/templates/**/*"))
+    };
+}
+
+/// Create a `Handler` which is invoked when responding to a `Request`.
+///
+/// How does a function become a `Handler`?.
```

Comment:
Maybe replace these last few lines with information about how you're using Tera, and what it's rendering.